transport/src/main/java/com/strategyobject/substrateclient/transport/RpcBoolean.java

```diff
@@ -0,0 +1,67 @@
+package com.strategyobject.substrateclient.transport;
+
+import lombok.RequiredArgsConstructor;
+
+import java.util.List;
+import java.util.Map;
+
+@RequiredArgsConstructor
+public class RpcBoolean implements RpcObject {
+
+    private final boolean value;
+
+    @Override
+    public boolean isNull() {
```

Comment:
I would propose to make each method returns `false` by default and throws `IllegalStateException` appropriately at `RpcObject` and then just override methods that relate to the 
particular implementation.